README.md

```diff
@@ -72,6 +72,19 @@ Optionaly you can specify path to server cert using `postgresql_cert_path` varia
 ```ymal
 postgresql_cert_path: "/etc/pki/tls/certs"
 ```
+### postgresql_certificates
+This is a `list` of `dict` in the same format as used
+  by the `fedora.linux_system_roles.certificate` role.  Specify this variable if
+  you want the certificate role to generate the certificates for the logging system
+  configured by the logging role. With this example, `self-signed` certificate
+  `logging_cert.crt` is generated in `/etc/pki/tls/certs`.
+  Default to `[]`.
+```yaml
+postgresql_certificates:
+  - name: postgresql_cert
```

Comment:
Is this value the same as the value of `postgresql_cert_name`?

Reply:
thanks for this point, it was the same value. I've made changes in the system role to be enough to specify it only once.